constructor/_schema.py

```diff
@@ -242,6 +242,21 @@ class ConstructorConfiguration(BaseModel):
 
     At least one channel must be supplied, either in `channels` or `channels_remap`.
     """
+    mirrored_channels: dict[NonEmptyStr, list[NonEmptyStr]] = {}
+    """
+    A mapping of channels to their mirror URLs. Each channel maps to a list of
+    mirror URLs that will be used as fallbacks. Example use:
+
+    ```yaml
+    mirrored_channels:
+      conda-forge:
+        - "https://conda.anaconda.org/conda-forge"
+        - "https://conda.anaconda.org/conda-forge-mirror"
+      defaults:
+        - "https://repo.anaconda.com/pkgs/main"
+        - "https://repo.anaconda.com/pkgs/main-mirror"
+    ```
```

Comment:
```suggestion
    A mapping of channels to their mirror URLs. Each channel maps to a list of
    mirror URLs that will be used as fallbacks.
    
    Requires `mamba` in the `base` environment.
    
    Example use:

    ```yaml
    mirrored_channels:
      conda-forge:
        - "https://conda.anaconda.org/conda-forge"
        - "https://conda.anaconda.org/conda-forge-mirror"
      defaults:
        - "https://repo.anaconda.com/pkgs/main"
        - "https://repo.anaconda.com/pkgs/main-mirror"
    ```
```

Let's mention this important detail!

Reply:
And, is `write_condarc: True` required? If so, include a mention to that too, thanks!

Reply:
Done!